efel/cppcore/LibV5.cpp

```diff
@@ -2400,20 +2398,20 @@ static int __sag_time_constant(const vector<double>& times,
   // minimal required length of each decay (indices)
   size_t min_length = 10;
 
-  // get start index
-  const size_t decayStartIdx =
-      distance(voltage.begin(),
-          find_if(voltage.begin(), voltage.end(),
-                  std::bind2nd(std::less_equal<double>(), minimum_voltage)));
+    // get start index
+    const size_t decayStartIdx =
+      std::distance(voltage.begin(),
```

Comment:
there are some more std::distance here

Reply:
The previous commit was for LibV1, the next ones repeat it for LibV5 for std::distance and std::find_if.